microsoft-365/compliance/audit-standard-setup.md

```diff
@@ -27,7 +27,7 @@ description: "This article describes how to set up Audit (Standard) so you can s
 
 # Set up Microsoft Purview Audit (Standard)
 
-Microsoft Purview Audit (Standard) in Microsoft 365 lets you search for audit records for activities performed in the different Microsoft 365 services by users and admins. Because Audit (Standard) is enabled by default for most Microsoft 365 and Office 365 organizations, there's only a few things you need to do before you, and others in your organization can search the audit log.
+Microsoft Purview Audit (Standard) in Microsoft 365 lets you search for audit records for activities performed in the different Microsoft 365 services by users and admins. Because Audit (Standard) is enabled by default for Microsoft 365 and Office 365 organizations, there's only a few things you need to do before you, and others in your organization can search the audit log.
```

Comment:
```suggestion
Microsoft Purview Audit (Standard) in Microsoft 365 lets you search for audit records for activities performed in the different Microsoft 365 services by users and admins. Because Audit (Standard) is enabled by default for Microsoft 365 and Office 365 organizations, there are only a few things you need to do before you, and others in your organization can search the audit log.
```